src/components/app/layout/AppHeader.tsx

```diff
@@ -71,7 +73,14 @@ const AppHeader: FC<AppHeaderProps> = ({ noBackground = false }) => {
                 )}
 
                 <Box pl={10}>
-                    <VolumeControl />
+                    {/* For backwards compatability, tolerate absense of the actions list */}
```

Comment:
How much do you care about backwards compatibility?

Reply:
Yeah, at first I put these buttons inside the existing `<VolumeControl>`. But that's a far more complicated component - and so the simple buttons just seemed lost amidst the sea of handling the popover and volume arc. But I'm sure it could be improved further :)

Reply:
Agreed. Keeping it how it is in this PR makes perfect sense.